.github/workflows/ci.yml

```diff
@@ -49,8 +53,17 @@ jobs:
           ini-values: zend.assertions=1, error_reporting=-1, display_errors=On
           coverage: xdebug
 
+      - name: Lock to a specific PHPUnit version
+        if: matrix.phpunit
+        run: composer require --no-update --dev phpunit/phpunit:${{ matrix.phpunit }}
+
       - name: Install Composer dependencies
         run: composer install --ansi --no-interaction --no-progress
 
+      - name: Migrate PHPUnit config to avoid warnings
```

Comment:
The migration seems to affect PHPUnit 10, not 9:
```
$ vendor/bin/phpunit --migrate-configuration                 
PHPUnit 9.6.6 by Sebastian Bergmann and contributors.

.../prophecy-phpunit/phpunit.xml.dist does not need to be migrated.
```
With PHPUnit 10, it results in this specific change:
```diff
   </testsuites>
-  <coverage>
+  <coverage/>
+  <source>
     <include>
       <directory>./src</directory>
     </include>
-  </coverage>
+  </source>
 </phpunit>
```

Reply:
This change looks wrong to me. Even in the 10.0 branch, the XSD still uses `coverage`. I would say that this config migrator is buggy.

Reply:
The migrator is not buggy, it's following https://github.com/sebastianbergmann/phpunit/issues/5294

The issue here is that PHPUnit outputs a warning to push toward the migration, and that's messing up with the tests: https://github.com/phpspec/prophecy-phpunit/actions/runs/4619041764/jobs/8167251236#step:6:26

The only other solution would be to suppress that diff in the PHPT tests with a `%a`, but it would probably make the test less reliable. Do you want me to do that?

Reply:
Actually, I don't think the phpunit commands launched by the tests themselves should read the phpunit.xml file of the project itself. And then, the would not have a coverage configuration at all that would mess with this config.

Reply:
the run_tests helper should add the `--no-configuration` flag

Reply:
Done in 76c13a9